helmfile.d/snippets/env.gotmpl

```diff
@@ -1,9 +1,10 @@
-{{- $teams := readFile "../env/env/teams.yaml" | fromYaml }}
-{{- $settings := readFile "../env/env/settings.yaml" | fromYaml }}
+{{- $ENV_DIR := env "ENV_DIR" }}
+{{- $teams := readFile (print $ENV_DIR "/env/teams.yaml") | fromYaml }}
+{{- $settings := readFile (print $ENV_DIR "/env/settings.yaml") | fromYaml }}
 {{- $teams := keys $teams.teamConfig.teams}}
-{{- $hasSops := eq (exec "bash" (list "-c" "( test -f ../env/.sops.yaml && echo 'true' ) || echo 'false'")) "true" }}
-{{- $charts := (exec "bash" (list "-c" "find  ../env/env/charts -name '*.yaml' -not -name 'secrets.*.yaml'")) | splitList "\n" }}
-{{- $chartsSecret := (exec "bash" (list "-c" "find  ../env/env/charts -name 'secrets.*.yaml'")) | splitList "\n" }}
+{{- $hasSops := eq (exec "bash" (list "-c" "( test -f $ENV_DIR/.sops.yaml && echo 'true' ) || echo 'false'")) "true" }}
+{{- $charts := (exec "bash" (list "-c" "find  $ENV_DIR/env/charts -name '*.yaml' -not -name 'secrets.*.yaml'")) | splitList "\n" }}
```

Comment:
I don't fully understand, when I came up with a proposal to do it like this you told me it wasn't a good idea, so I wonder what has changed?

Reply:
and I doubt that you have tested it
```
2021-12-14T10:24:44.462Z otomi:global:error Error: in /home/app/stack/helmfile.tpl/helmfile-dump-files.yaml: error during ../helmfile.d/snippets/env.gotmpl.part.0 parsing: template: stringTemplate:2:14: executing "stringTemplate" at <readFile (print $ENV_DIR "/env/teams.yaml")>: error calling readFile: open /env/teams.yaml: no such file or directory
--
5 | at hfCore (/home/app/stack/dist/src/common/hf.js:46:30)
6 | exit code: 1
7 | at ChildProcess.<anonymous> (/home/app/stack/node_modules/zx/dist/bundle.cjs:15343:22)
8 | at ChildProcess.emit (node:events:390:28)
9 | at maybeClose (node:internal/child_process:1064:16)
10 | at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
```

Reply:
> and I doubt that you have tested it

I installed the latest `secrets-generation` tag and saw no problems. I am betting on the tests to cover edge cases. If we found another one we should add a test for that.

Reply:
It happens during the team deployment.

Reply:
it seems otomi.ts is not setting ENV_DIR anymore...it was before I believe

Reply:
I see now. We resolve env now late with `env()` and so `otomi.ts` does not see the default set with envalid

Reply:
> I don't fully understand, when I came up with a proposal to do it like this you told me it wasn't a good idea, so I wonder what has changed? (referring to using `$ENV_DIR` as opposed to `../env/env..`)

Don't remember exactly. I do know I need it to be able to debug otomi commands targeting the cluster. It will keep working with IN_DOCKER=1 still